### PR TITLE
Very basic timer.

### PR DESCRIPTION
**Pull request by wouterz**
Also used final variable for health.

Timer starts increasing as soon as spaceship object is created.

### CODE REVIEW

**Comment by jessetilro** [comment]
The Route is already tracking the progress using the timestamp and duration fields, so I think this might be redundant :(

**Comment by wouterz** [comment]
my vote would go use both and check the timer against the duration. Time elapsed seems more of an attribute of the spaceship and the timer will be easier to mutate for possible future implementation such as pausing the game, bonus/penalty times, etc

File: src/main/java/pixelperfect/spaceship/Spaceship.java [hunk 1]
```diff
@@ -11,17 +12,21 @@
  */
 public class Spaceship {
 
+  private static final int STARTING_HEALTH = 100;
```
Comment (FlorisXll) [inline src/main/java/pixelperfect/spaceship/Spaceship.java]: Note: we'll probably make a data class later I think.

**Comment by jessetilro** [comment]
In that case we should remove the Route's timestamp and duration, for they are tracking exactly the same data as the timer introduced in the Spaceship, it really is redundant to have both. However, the spaceship then has to notify the route occasionally of the progress, since only the Route knows when to introduce new RouteNode's (points along the route where the crew will have to make a decision which way to go). Looks good to me for now.

**Comment by dmalarev** [comment]
looks good to me so far

**Pull request merged**